torch/ao/quantization/quantizer/x86_inductor_quantizer.py

```diff
@@ -654,6 +654,8 @@ def _annotate_conv2d_fusion_pattern(
         self._annotate_conv2d_binary(model, config)
         self._annotate_conv2d_unary(model, config)
         self._annotate_conv2d(model, config)
+        self._annotate_linear_binary_unary(model, config)
```

Comment:
I think we also need the recipe for QAT.

Reply:
Looks like we don't have QAT annotator for linear. How about adding them later.

Reply:
Sure, I think we need it.

Reply:
Sure